docs/scripting-ref.rst

```diff
@@ -331,6 +331,32 @@ Compare:
 See also: :ref:`splash-runjs`, :ref:`splash-jsfunc`,
 :ref:`splash-wait-for-resume`, :ref:`splash-autoload`.
 
+.. _splash-disablejs:
+
+splash:disablejs
```

Comment:
The docs are incorrect, by the way: methods are named enable_js/disable_js in the code, but enablejs/disablejs in the docs. Also, underline is too short.